src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -41,6 +41,7 @@ export function CreateExportModal({
   const [issuesToExport, setIssuesToExport] = useState<Issue[]>([]);
   const [startDate, setStartDate] = useState<Date | null>(null);
   const [endDate, setEndDate] = useState<Date | null>(null);
+  const [hoveredoverexport, sethoveroverexport] = useState(false);
```

Comment:
```suggestion
  const [exportHovered, setExportHovered] = useState(false);
```
Better variable name + camel casing